docs/ops/arithmetic/Tan_1.md

```diff
@@ -6,32 +6,39 @@
 
 **Short description**: *Tan* performs element-wise tangent operation with given tensor.
 
-**Attributes**:
+**Detailed description**:  Operation takes one input tensor and performs the element-wise tangent function on a given input tensor, based on the following mathematical formula:
 
-    No attributes available.
+\f[
+a_{i} = tan(a_{i})
+\f]
+
+*Example 1*
+
+    input = [0.0, 0.25, -0.25, 0.5, -0.5]
+    output = [0.0, 0.25534192, -0.25534192, 0.54630249, -0.54630249]
+
+*Example 2*
+
+    input = [-2, -1, 0, 1, 2]
+    output = [2, -2, 0, 2, -2]
+
+**Attributes**: *tan*  operation has no attributes.
 
 **Inputs**
 
-* **1**: An tensor of type *T*. **Required.**
+* **1**: A tensor of type *T* and arbitrary shape, measured in radians. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise tan operation. A tensor of type *T*.
+* **1**: The result of element-wise *tan* applied to the input tensor. A tensor of type *T* and same shape as the input tensor.
 
 **Types**
 
-* *T*: any numeric type.
+* *T*: int32,int64,uint32,uint64,float16,float32
```

Comment:
We do not specify types like this. As plugins may support different types. Please change this to -> \*T\*: any supported numeric type.

Reply:
Update the document.  Are the test case about uint ok ?